### DISCUSSION TITLE
Can we create a way coolify can have  a domain name setup instead of using ip address  in the  login and setup of the server

### DISCUSSION THREAD

**Discussion by lewismunene020** [coollabsio/coolify#2051] [Feature Requests]
I started working  with  coolify lately and i see alot  of potential in it  .But  is there a way to use a domain name instead  of   ip address  to manage everything in an encrypted way   ?

**Comment by jsmits** [comment]
Yes, just create a DNS entry (an A record) for your domain pointing to your Coolify server IP and put that in Settings -> General -> Instance’s Domain.

**Reply by lewismunene020** [reply]
thanks  i managed to set it  up

**Reply by suffianali0** [reply]
i did the same way but it say the connection to this page is not secure

**Reply by lewismunene020** [reply]
Restart your proxy that may help
